src/main/java/com/google/sps/data/PossibleInterviewer.java

```diff
@@ -24,6 +24,12 @@
   public abstract String job();
 
   public static PossibleInterviewer create(String company, String job) {
+    if (company.equals("")) {
```

Comment:
Could we do this on the JavaScript side instead? I think adding the fake values in JavaScript is a bit dangerous in case this object was ever used for a different purpose (i.e. in a way that could cause the "<Not specified>" to be written back to DataStore).

Reply:
This object is currently only used for generating the front end, what did you have in mind with javascript?

Reply:
I'll change it so there's a 'could-be-not-specified' class or something and change the text with a javascript function call before the text is displayed.